package.json

```diff
@@ -4,5 +4,4 @@
   "description": "Data files for Digital Marketplace’s procurement frameworks",
   "repository": "git@github.com:alphagov/digitalmarketplace-frameworks",
   "author": "enquiries@digitalmarketplace.service.gov.uk",
```

Comment:
I assume NPM's JSON parser is happy with this trailing `,`?

Reply:
It is not 😢